### PR TITLE
feat: sync related ui alerts on pool page

### PR DESCRIPTION
**Pull request by alter-eggo**
# Description
Add cross chain boost sync ui alerts on pool page

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Review by garethfuller — CHANGES_REQUESTED** [review]
Add a close modal button to the bottom of this modal, button text can just be "Close". cc @pkattera 

Screenshot 2023-06-27 at 10 24 21

**Comment by pkattera** [comment]
> I recently staked to Arbitrum, on the veBAL page it says Arbitrum is synced, you can see my veBAL balance is being displayed here on my Arbitrum portfolio. But on my Arbitrum portfolio page it doesn't suggest that I have any boost or that I should re-stake those positions. Is it a bug or am I waiting for something else? If so we need to communicate that. cc @pkattera

Agree. This 'restake' warning should appear @alter-eggo 

CleanShot 2023-06-27 at 10 38 29@2x

File: src/components/_global/BalAccordion/BalAccordion.vue [hunk 1]
```diff
@@ -17,10 +17,12 @@ type Props = {
   dependencies?: unknown;
   showSectionBorder?: boolean;
   reCalcKey?: number;
+  isOpenedByDefault?: boolean;
```
Comment (garethfuller) [inline src/components/_global/BalAccordion/BalAccordion.vue:20]: This didn't work for me.

File: src/components/contextual/pages/vebal/cross-chain-boost/StakingCardSyncAlert.vue [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      shouldShowWarningAlert.value = true;
+      emit('shouldStakingCardBeOpened');
+    }
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+onBeforeMount(() => {
+  void setWarningAlertState();
```
Comment (garethfuller) [inline src/components/contextual/pages/vebal/cross-chain-boost/StakingCardSyncAlert.vue:85]: void?

File: src/components/contextual/pages/vebal/cross-chain-boost/StakingCardSyncAlert.vue [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+const tipText = computed(() => {
+  if (hasNonPrefGaugeBalance.value) {
+    const title = 'Restake to be eligible for future incentives';
+    let text = '';
+    if (
+      networksSyncState.value[networkId.value] === NetworkSyncState.Unsynced
+    ) {
+      text =
+        'After your veBAL syncs to [Arbitrum], restake to move from the deprecated pool gauge to the new boost-aware pool gauge. If you restake before the veBAL syncs, you’ll need to perform another interaction on the gauge to start receiving your staking boost.';
```
Comment (garethfuller) [inline src/components/contextual/pages/vebal/cross-chain-boost/StakingCardSyncAlert.vue:34]: Should this text be changed? `[Arbitrum]`?

File: src/pages/vebal.vue [hunk 4]
```diff
@@ -21,7 +21,7 @@ providePoolStaking();
   <div>
     <Hero v-if="isVeBalSupported" v-once />
 
-    <div class="py-16 xl:py-20 bg-gray-50 dark:bg-gray-900">
+    <div class="py-16 xl:py-20 bg-gray-50 dark:bg-gray-850/50">
```
Comment (garethfuller) [inline src/pages/vebal.vue:24]: Why change this?

File: src/providers/cross-chain-sync.provider.ts [hunk 5]
```diff
@@ -267,6 +280,38 @@ export const crossChainSyncProvider = () => {
...
+  async function triggerGaugeUpdate(gaugeAddress: string) {
+    const gaugeContract = new LiquidityGauge(gaugeAddress);
+
+    const signer = getSigner();
+    const tx = await gaugeContract.checkpointUser({
+      signer,
+      userAddress: account.value,
+    });
+
+    return tx;
```
Comment (garethfuller) [inline src/providers/cross-chain-sync.provider.ts:312]:
```suggestion
    return gaugeContract.checkpointUser({
      signer,
      userAddress: account.value,
    });
```

File: src/providers/cross-chain-sync.provider.ts [hunk 6]
```diff
@@ -267,6 +280,38 @@ export const crossChainSyncProvider = () => {
...
+      contractAddress
+    );
+    const balances =
+      await workingBalanceHelperContract.getWorkingBalanceToSupplyRatios({
+        signer,
+        userAddress: account.value,
+        gauge: gaugeAddress,
+      });
+
+    return balances;
```
Comment (garethfuller) [inline src/providers/cross-chain-sync.provider.ts:299]:
```suggestion
    return workingBalanceHelperContract.getWorkingBalanceToSupplyRatios({
        signer,
        userAddress: account.value,
        gauge: gaugeAddress,
      });
```

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: src/pages/vebal.vue (hunk 4 above)
Comment (alter-eggo) [reply src/pages/vebal.vue:24]: @pkattera

Comment (pkattera) [reply src/pages/vebal.vue:24]: If you look at light mode, you can see contrast vs this section and the one below. 

On dark mode, the veBAL management section and the voting table have the same background color.

Without this change, there is a really awkward bottom margin distance between the bottom of the veBAL management section and the voting table. I felt this approach using opacity was the easiest way to establish some contrast within the available color space.

File: src/components/contextual/pages/vebal/cross-chain-boost/StakingCardSyncAlert.vue (hunk 2 above)
Comment (alter-eggo) [reply src/components/contextual/pages/vebal/cross-chain-boost/StakingCardSyncAlert.vue:85]: why not?

Comment (garethfuller) [reply src/components/contextual/pages/vebal/cross-chain-boost/StakingCardSyncAlert.vue:85]: Why is it needed?

Comment (alter-eggo) [reply src/components/contextual/pages/vebal/cross-chain-boost/StakingCardSyncAlert.vue]: hm, yeah, this is async function and I was feeling like `void` should be there. removed

File: src/components/_global/BalAccordion/BalAccordion.vue (hunk 1 above)
Comment (alter-eggo) [reply src/components/_global/BalAccordion/BalAccordion.vue:20]: fixed

**Pull request merged**